main.py

```diff
@@ -25,6 +25,28 @@ def draw_debug_output(user_stats_tracker: UserStatsTracker) -> None:
         st.write(f"- active user ID: {user_id}, Status: {user_data.status}")
 
 
+def create_button(
+    current_status: UserStatus,
+    button_status: UserStatus,
+    user_stats_tracker: UserStatsTracker,
+) -> None:
+    if current_status == button_status:
+        st.button(
+            button_status.value,
+            use_container_width=True,
+            type="primary",  # Highlight button if selected
```

Comment:
whats your opionon on comments? 
or is this just AI code? maybe we could disable comment generation in agends.md 

T.b.d.